### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2245]
It's a link with no text, an image with no alt text. Thazzit.

**Comment by rebeccacremona** [comment]
Okay, this appears to be a Safari bug: though we are providing an empty alt string, on purpose, to suppress any weird screen-reader attempts to describe the image (here, "Empty Layers"), Safari is stripping the string.

Safari:
![image](https://user-images.githubusercontent.com/11020492/36326236-5cf1f51e-1328-11e8-9df0-467e2781bed9.png)

Firefox:
![image](https://user-images.githubusercontent.com/11020492/36326253-67f03426-1328-11e8-812b-280200ed4524.png)

That's not how it's supposed to work.

I'm going to suggest that going forward, we simply add a redundant aria-hidden="true" whenever we also have a blank alt text (it can't hurt).

**Comment by rebeccacremona** [comment]
In any case, I think it's kind of neat that the infinity symbol is our logo: it's appropriate to know that, even if you can't see it, so, for a change, I'm going to add an alt text for a logo.
